src/meta/play-meta-util.js

```diff
@@ -1,98 +1,45 @@
+const plays = () => [];
 
-import { plays } from './play-meta';
-
-const getAllPlays = () => {
-  return plays;
-};
-
-const getPlayById = id => {
-  return plays.find(play => play.id === id);
+const getPlayById = (id) => {
+  return plays().find((play) => play.id === id);
 };
 
-const getPlaysOnSearch = searchTerm => {
-  return plays.filter(play => {
-    return (play.name.toLowerCase().includes(searchTerm.toLowerCase())
-      || play.description.toLowerCase().includes(searchTerm.toLowerCase()));
-  });
-}
-
-const getPlaysByTags = tags => {
-  return plays.filter(play => {
-    return play.tags.includes(tags);
-  });
-}
-
-const getPlaysByLevel = level => {
-  return plays.filter(play => {
-    return play.level === level;
+const getPlaysOnSearch = (searchTerm) => {
+  return plays().filter((play) => {
+    return (
+      play.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
+      play.description.toLowerCase().includes(searchTerm.toLowerCase())
+    );
   });
-}
-
-const getPlaysByCreator = creator => {
-  return plays.filter(play => {
-    return play.github === creator;
-  });
-}
-
-const getPlaysByLanguage = language => {
-  return plays.filter(play => {
-    const lang = play.language || 'js';
-    return lang === language;
-  });
-}
+};
 
-const getAllTags = () => {
+const getAllTags = (plays) => {
   const tags = plays.reduce((acc, play) => {
-    return acc.concat(play.tags.split(','));
+    return acc.concat(
+      play.play_tags.reduce((pre, next) => {
+        return pre.concat({tag: next.tag.name, id: next.tag.id});
+      }, [])
+    );
   }, []);
-
-  return Array.from(new Set([...tags]));
-}
-
-const getAllCreators = () => {
-  const creators = plays.reduce((acc, play) => {
-    play.github && acc.push(play.github);
-    return acc;
-  }, []);
-
-  return Array.from(new Set([...creators]));
-}
-
-const getAllLevels = () => {
-  const levels = plays.reduce((acc, play) => {
-    return acc.concat(play.level);
-  }, []);
-
-  return Array.from(new Set([...levels]));
-}
-
-const getAllLanguages = () => {
-  const languages = plays.reduce((acc, play) => {
-    const lang = play.language || 'js';
-    return acc.concat(lang);
-  }, []);
-
-  return Array.from(new Set([...languages]));
-}
+  let finalArr = new Array();
+  tags.forEach((item) => {
+    if (!finalArr.find(_ => _.id === item.id)) {
+      finalArr.push(item);
+    }
+  });
+  return finalArr;
+};
 
 const getFeaturedPlays = () => {
-  const featuredPlays = plays.filter(play => {
+  const featuredPlays = plays().filter((play) => {
```

Comment:
Do we need it anymore?

Reply:
no most. most of the staff isnt needed in play-meta-util. i kept coz filtering wast working well. anyway will remove.